README.md

```diff
@@ -22,12 +22,16 @@ See [https://docs.msupply.foundation/en:cold_chain:start](https://docs.msupply.f
 - Install Android Studio and SDK tools: https://developer.android.com/studio.
 - Install SDKMAN for managing Java versions: https://sdkman.io/.
 
+The application is using java v1.8 you may need to target a specific java version in order to run the app locally.
```

Comment:
this is because I had java 17 running and had to downgrade in order to `yarn start`